deployments/helm-chart/templates/controller-hpa.yaml

```
@@ -0,0 +1,37 @@
+{{- if and .Values.controller.autoscaling.enabled (eq .Values.controller.kind "deployment") (semverCompare ">=1.23.0" .Capabilities.KubeVersion.Version) -}}
```

```suggestion
{{- if and .Values.controller.autoscaling.enabled (eq .Values.controller.kind "deployment") (semverCompare ">=1.21.0" .Capabilities.KubeVersion.Version) -}}
```

This should be 1.21?